Jenkinsfile

```diff
@@ -40,6 +40,6 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-// centos8: No matching package to install: 'fio-devel'
-packageBuildingPipelineDAOS(['distros': ['centos7', 'centos8', 'leap15',
+// el8: No matching package to install: 'fio-devel'
```

Comment:
Since we are now building on `el8`, this comment must be stale.  It can be removed if you would please.